client/src/App.jsx

```diff
@@ -8,10 +8,12 @@ import Navbar from "./Components/CommonComponent/Navbar";
 import Profile from "./Components/Layouts/SingleCustomer";
 
 import Login from "./Components/Layouts/Login";
+import Ordersmanagement from "./Components/Layouts/Ordersmanagement/index";
```

Comment:
@Anies12 This looks like a duplicate import (with a slightly different name) so you can probably delete this line.